src/main/java/mate/jdbc/util/ConnectionUtil.java

```diff
@@ -6,10 +6,10 @@
 import java.util.Properties;
 
 public class ConnectionUtil {
-    private static final String URL = "YOUR DATABASE URL";
-    private static final String USERNAME = "YOUR USERNAME";
-    private static final String PASSWORD = "YOUR PASSWORD";
-    private static final String JDBC_DRIVER = "YOUR DRIVER";
+    private static final String URL = "jdbc:mysql://localhost:3306/taxi_db";
```

Comment:
You created DB `taxi_service` in file init_db.sql
```suggestion
    private static final String URL = "jdbc:mysql://localhost:3306/taxi_service";
```